setup/dns.sh

```diff
@@ -113,3 +113,4 @@ chmod +x /etc/cron.daily/mailinabox-dnssec
 
 ufw_allow domain
 
+restart_service nsd
```

Comment:
I must remember the exact reason of this, but I think the first NSD start creates a configuration file that is required by a later script ... I can't remember which script was really involved, but removing this will cause problems.